apic_ml2/neutron/plugins/ml2/drivers/cisco/apic/mechanism_apic.py

```diff
@@ -489,6 +529,16 @@ def _setup_opflex_rpc_listeners(self):
             self.opflex_topic, self.opflex_endpoints, fanout=False)
         return self.opflex_conn.consume_in_threads()
 
+    def _setup_keystone_notification_listeners(self):
```

Comment:
No its not working as notification is a completely different mechanism than the rpc calls.... I was trying that first in the hope that I can just re-use the code but that's not the case.